baseplate/sidecars/live_data_watcher.py

```diff
@@ -32,6 +36,41 @@ def __init__(self, dest: str, owner: int, group: int, mode: int):
         self.group = group
         self.mode = mode
 
+    @staticmethod
+    def fetch_data_from_url(url: str) -> Optional[str]:
+        data = None
+        try:
+            # Fetch the data from the url as bytes.
+            data = requests.get(url).content
+        except requests.exceptions.RequestException as e:
+            logger.exception(e)
+            return None
+        return data
+
+    @staticmethod
+    def get_data_to_write(json_data: dict, data: bytes) -> Optional[bytes]:
+        # Check if we have a JSON in this special format:
+        # data = {
+        #    "live_data_watcher_load_type": str
```

Comment:
I think we actually want to support a `raw` (or whatever it's called) type in addition to `http`.

the case is that if some service actually need to sync a json file down via live data fetcher with `"live_data_watcher_load_type": "http"` field, they can wrap it again with `raw` and put the whole json blob into the top level `data` field.

Reply:
@fishy I agree. This is why I included the `live_data_watcher_load_type` key so that anyone in the future could add something like `raw`. Did you want me to support this in this PR itself? I can add that if you want. I just wanted to limit the scope to what the Experiments team needed (url), but have no issue with adding it.

Reply:
Updated to new approach.